### PR TITLE
Added events for CL Join example.

### PR DESCRIPTION
**Pull request by d-stahl-ericsson**
A JSON array of events included in the Confidence Level Joining Example.

### CODE REVIEW

File: flows/confidence-level-joining/events.json [hunk 1]
```diff
@@ -0,0 +1,343 @@
...
+  {
+    "meta": {
+      "type": "EiffelConfidenceLevelModifiedEvent",
+      "version": "1.0",
+      "time": 21000,
+      "domainId": "example.domain",
+      "id": "aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeee18"
+    },
+    "data": {
+      "name": "functionalComponentTestsPassed",
```
Comment (p-backman-ericsson) [inline flows/confidence-level-joining/events.json:325]: Shouldn't the name be functionalComponentTests ?

File: flows/confidence-level-joining/events.json [hunk 2]
```diff
@@ -0,0 +1,343 @@
...
+      }
+    },
+    "links": {
+      "causes": [
+        "aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeee13",
+        "aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeee14",
+        "aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeee15",
+        "aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeee16"
+      ],
+	  "subjects": [
```
Comment (p-backman-ericsson) [inline flows/confidence-level-joining/events.json]: There is a tab messing up the indentation here.

File: flows/confidence-level-joining/events.json (hunk 1 above)
Comment (d-stahl-ericsson) [inline flows/confidence-level-joining/events.json:325]: Does it really matter? I think it's a minor detail, but "...Passed" actually makes more sense to me. We think of confidence levels as a kind of stamp, signaling that a certain set of criteria has been achieved, correct?

File: flows/confidence-level-joining/events.json (hunk 2 above)
Comment (d-stahl-ericsson) [inline flows/confidence-level-joining/events.json]: Thanks! Those tabs have been annoying me to no end. I finally managed to change the editor settings, so it shouldn't be a problem anymore.

File: flows/confidence-level-joining/events.json (hunk 1 above)
Comment (p-backman-ericsson) [inline flows/confidence-level-joining/events.json:325]: I think that the combination of name and value is strange. I interpret "Passed" as a value and it is now in the name. I am reading TestsPassed = Failure which sound funny to me. But it is not a major concern.

**Pull request merged**